components/Card.vue

```diff
@@ -144,9 +135,10 @@ export default Vue.extend({
       }
     }
   },
-  methods: {
-    imageLink(id: string) {
-      return `${this.$axios.defaults.baseURL}/Items/${id}/Images/Primary`;
+  mounted(): void {
+    if (this.item.ImageTags.Primary) {
+      const button = this.$refs.cardButton as HTMLElement;
+      button.style.backgroundImage = `url(${this.$axios.defaults.baseURL}/Items/${this.item.Id}/Images/Primary?maxHeight=${button.clientHeight}&maxWidth=${button.clientWidth}&tag=${this.item.ImageTags.Primary}&quality=90)`;
```

Comment:
Would it not be better to move this to the imageHelper mixin. From there, pass in a set of options relating to height/width/tag?

Reply:
I did a relatively simple version for now. I'll refactor it properly in a subsequent PR, to limit the changes made in this one.